pkg/channel/history_transformer.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package channel
+
+import (
+	"github.com/cloudevents/sdk-go/pkg/binding"
```

Comment:
try to see if you can use just the alias and import only `"github.com/cloudevents/sdk-go"`

Reply:
I can't because i miss TransformerFactory. I'm fine keeping the binding imports

Reply:
take note and make a PR in the cloudevents side I was doing that yesterday too